src/api/procedures/controllerTransfer.ts

```diff
@@ -59,28 +65,51 @@ export async function prepareControllerTransfer(
     {},
     stringToTicker(ticker, context),
     numberToBalance(amount, context),
-    portfolioIdToMeshPortfolioId(portfolioId, context)
+    portfolioIdToMeshPortfolioId(originPortfolioId, context)
   );
 }
 
 /**
  * @hidden
  */
 export function getAuthorization(
-  this: Procedure<Params, void>,
+  this: Procedure<Params, void, Storage>,
   { ticker }: Params
 ): ProcedureAuthorization {
+  const {
+    storage: { originPortfolioId },
+    context,
+  } = this;
   return {
-    identityRoles: [{ type: RoleType.TokenPia, ticker }],
+    identityRoles: [
+      { type: RoleType.TokenPia, ticker },
+      { type: RoleType.PortfolioCustodian, portfolioId: originPortfolioId },
```

Comment:
Not the origin portfolio, the PIA's default portfolio. Same for the signer permissions below